electron-builder.json

```diff
@@ -156,21 +156,29 @@
   "win": {
     "target": [
       "nsis",
-      "zip"
+      "zip",
+      "msi"
     ],
     "extraFiles": [
       {
         "filter": [
           "LICENSE.txt",
           "NOTICE.txt"
         ]
+      },
+      {
+        "from": "resources/windows/gpo",
```

Comment:
All builds will now ship with the GPO artifacts, since it's only helpful to those that need them and doesn't matter for those that don't.